backend/app.py

```diff
@@ -481,6 +492,7 @@ def parse_codeset_ids(qstring):
     requested_codeset_ids = [int(x) for x in requested_codeset_ids]
     return requested_codeset_ids
 
+# Routes ---------------------------------------------------------------------------------------------------------------
```

Comment:
I was having trouble understanding the major sections of `app.py`, so I added these commented section labels / dividers. Eventually this will be easier to read when we move to RDBMS and also refactor to put routes in separate file(s), etc.